### PR TITLE
Makes space cleaner more efficient, reduces spraybottle volume

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
:cl: Naksu
balance: The space cleaner spray bottle is now much more efficient and uses much less space cleaner per spray. The amount of cleaner it can hold has been adjusted to compensate.
/:cl:

### CODE REVIEW

**Comment by arsserpentarium** [comment]
it's nerf.not buff

File: code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -147,11 +147,13 @@
 /obj/item/reagent_containers/spray/cleaner
 	name = "space cleaner"
 	desc = "BLAM!-brand non-foaming space cleaner!"
-	list_reagents = list("cleaner" = 250)
+	list_reagents = list("cleaner" = 50)
+	amount_per_transfer_from_this = 1
+	stream_amount = 5
```
Comment (PKPenguin321) [inline code/modules/reagents/reagent_containers/spray.dm]: bruh. look at this duuude

File: code/modules/reagents/reagent_containers/spray.dm [hunk 2]
```diff
@@ -20,7 +20,7 @@
 	var/stream_amount = 10 //the amount of reagents transfered when in stream mode.
 	var/can_fill_from_container = TRUE
 	amount_per_transfer_from_this = 5
-	volume = 250
```
Comment (PKPenguin321) [inline code/modules/reagents/reagent_containers/spray.dm]: wait till you see this, oh no no no no

File: code/modules/reagents/reagent_containers/spray.dm [hunk 3]
```diff
@@ -20,7 +20,7 @@
 	var/stream_amount = 10 //the amount of reagents transfered when in stream mode.
 	var/can_fill_from_container = TRUE
 	amount_per_transfer_from_this = 5
-	volume = 250
+	volume = 50
```
Comment (PKPenguin321) [inline code/modules/reagents/reagent_containers/spray.dm]: HHHHHHHHHHHHHHHHHAHAHAHAHAHAAAAAAAa
LOOK AT THE TOP OF HIS HEAD
LOOK AT HIS LIPS

**Comment by CameronWoof** [comment]
Even if this was a buff, and it's not, Janitors should be less reliant on space cleaner. We have holosigns and fancy mops and all this other stuff so that they'll actually clean stuff manually. 

The crew dynamic created by a janitor mopping up a well-traveled area and putting up signs is completely negated when they have easy access to tons of cleaner instead.

**Comment by Armhulen** [comment]
@PKPenguin321 summed it up perfectly

**Comment by Repukan** [comment]
Gud change

**Comment by arsserpentarium** [comment]
Why don't you want to do something good?Like make regular beakers not 50/100,but 60/120? Why?because 50 can be divided by 5,25 and 2, but 60 by 2,3,4,5,6,10,12,30. So it's much better for mixing chems.

I know, sometimes you have bad day. But there will be better day.
P.S.How long ago did you seen BS beakers?guess why?

**Comment by arsserpentarium** [comment]
Also 50 u is ridicolous, since even botany sprays have 100.

**Comment by Therapist1** [comment]
yeah, i really don’t see the point in this. definitely not a buff by any stretch of the word.

**Comment by AlexTheSysop42** [comment]
bluespace beakers are for grenades

**Comment by 81Denton** [comment]
Imo just properly name it a chemistry nerf instead of """efficiency tweak"""

**Comment by ExcessiveUseOfCobblestone** [comment]
chemistry isn't nerfed when 15u in a syringe kills just as well...

**Comment by ExcessiveUseOfCobblestone** [comment]
@arsserpentarium 

> 60 ml beaker

delete!!!!

**Comment by Singularbyte** [comment]
To be fair to this change, it is kinda silly how much much they contain for their size. You can literally fit the equivalent of almost 2 whole full size fuel tanks in the space of a cardboard box simply by filling it with these sprays since each one is worth 1/4 of a tank. Hell, each spray contains more than three and a half buckets worth of fluid.

**Comment by arsserpentarium** [comment]
1.ExcessiveUseOfCobblestone, well, syringes is't only chemistry delight.
2.what?Why delete?
3.Meanwhile bucket is 70 u.I think, bucket should be 300 u.But needs to be bulky.

**Comment by ExcessiveUseOfCobblestone** [comment]
But that shifts the discussion to "having lots of u is !!FUN!!" instead of "this ruins chemistry!!!"

I have never worked with a 60u beaker. I mean I probably have in reality but they are only marked for 50 and ingame there's not really a thing of random error (which you'd just be able to null with the chem master anyways *sigh*)

**Comment by arsserpentarium** [comment]
@ExcessiveUseOfCobblestone Nah, it won't break anything. You can make even 666 u beaker.

**Comment by ExcessiveUseOfCobblestone** [comment]
sorry I had a buggy moment and was talking about real life

**Comment by Mark9013100** [comment]
Misleading PR title.

**Comment by ma44** [comment]
https://github.com/yogstation13/yogstation/pull/2601 when?

File: code/modules/reagents/reagent_containers/spray.dm (hunk 3 above)
Comment (duncathan) [reply code/modules/reagents/reagent_containers/spray.dm]: why did you nerf all spraybottles? this is unnecessary, just change the space cleaner bottle.

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by AlexTheSysop42** [comment]
This would make it so you can't make death mix sprays or healing sprays anymore (sure, medical sprays exist, but they aren't instant).

**Comment by 81Denton** [comment]
Synthflesh sprays are

**Pull request merged**